packages/rollup-plugin-index-html/src/process-entry-html.js

```diff
@@ -16,15 +17,27 @@ function processEntryHtml(inputConfig) {
 
   const indexFolder = path.dirname(inputConfig.input);
   const indexHTMLString = fs.readFileSync(inputConfig.input, 'utf-8');
-  const resources = extractResources(indexHTMLString);
-  const modules = resources.jsModules.map(p => path.join(indexFolder, p));
+  const {
+    jsModules,
+    inlineScripts,
+    inlineImportMaps,
+    indexHTML: outputIndexHTML,
+  } = extractResources(indexHTMLString);
+  const input = jsModules.map(p => path.join(indexFolder, p));
+  if (inlineScripts.length) {
+    inlineScripts.forEach(script => {
+      const inlineScriptPath = path.join(indexFolder, `inline.${md5(script)}.js`);
```

Comment:
rollup will hash the final script output, you're now hashing the unbuilt content. I think we can just use the script's array index as id. (`inline-${i}.js`)

Reply:
fine by me